lib/cylc/batch_sys_handlers/at.py

```diff
@@ -50,6 +50,7 @@ class AtCommandHandler(object):
     REC_ERR_FILTERS = [
         re.compile("warning: commands will be executed using /bin/sh")]
     REC_ID_FROM_SUBMIT_ERR = re.compile(r"\Ajob\s(?P<id>\S+)\sat")
+    SUBMIT_CMD_ENV = {"shell": "/bin/sh"}
```

Comment:
Fixed. I don't know how it happened, but I am sure I put ALL CAPS in when I was testing it! (Not surprisingly, the test battery fails with lowercase.)